pkg/cmd/clients.go

```diff
@@ -105,15 +105,18 @@ func (cc *ClientCmd) CreateClientCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "client <name> <clientType iOS|cordova|android>",
 		Short: "create a mobile client representation in your namespace",
-		Long: `Sets up the representation of a mobile application in your namespace. This used to provide a mobile client context for various
-actions such as doing mobile client builds,
-mobile --namespace=myproject create client <name> <clientType>
-kubectl plugin mobile create client <name> <clientType>
-oc plugin mobile create client <name> <clientType>
-`,
+		Long: `Sets up the representation of a mobile application of the specified type in your namespace.
+This is used to provide a mobile client context for various actions such as creating, starting or stopping mobile client builds.
+
+The available client types are android, cordova and iOS. 
+
+A namespace must be specified by providing the --namespace flag or by setting the KUBECTL_PLUGINS_CURRENT_NAMESPACE environment variable.`,
```

Comment:
the env var is set by default when you use it as a plugin. Maybe something like:

``` when used standalone, the --namespace flag must be specified``` followed up with the examples that are currently there, I think this will be enough to make it clear